dluhc-component-library/src/api/api.ts

```diff
@@ -17,3 +18,17 @@ export const fetchDataset = async (geometry: Geometry) => {
 
 export const fetchDatasetList: () => Promise<DatasetResponse> = async () =>
   fetch(`${baseURL}/dataset.json`).then((response) => response.json());
+
+export const fetchEntities: (
```

Comment:
`Non-blocker` Probably should pull the useQuery to this level instead of having to define it in the components each time but not something to worry about in this PR until we start using them more